README.md

```diff
@@ -151,12 +184,11 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 If everything goes well, you should see output similar to the following showing that the Chef Habitat Builder on-prem services are loaded:
 
 ```output
-hab-sup(MN): The habitat/builder-datastore service was successfully loaded
-hab-sup(MN): The habitat/builder-router service was successfully loaded
-hab-sup(MN): The habitat/builder-api service was successfully loaded
-hab-sup(MN): The habitat/builder-api-proxy service was successfully loaded
-hab-sup(MN): The habitat/builder-originsrv service was successfully loaded
-hab-sup(MN): The habitat/builder-sessionsrv service was successfully loaded
+hab-sup(AG): The habitat/builder-datastore service was successfully loaded
```

Comment:
Why these changes?

Reply:
Oh, this change is simply due to the interesting fact that I am not seeing those same service names in my local Builder.

Specifically, I don't see `builder-originsrv` or `habitat/builder-sessionsrv`.
Thus, I based the changes off of the list of services that I do see with `hab svc status` and `journalctl | grep loaded`:

```
root@chef-builder:~# hab svc status
package                                        type        desired  state  elapsed (s)  pid   group
habitat/builder-api/8483/20190917211441        standalone  up       up     255440       7871  builder-api.default
habitat/builder-datastore/7795/20181018210336  standalone  up       up     255441       7800  builder-datastore.default
habitat/builder-minio/7764/20181006010221      standalone  up       up     255441       7820  builder-minio.default
habitat/builder-api-proxy/8496/20190924062940  standalone  up       up     255441       7841  builder-api-proxy.default
habitat/builder-memcached/7728/20180929144821  standalone  up       up     255441       7851  builder-memcached.default
root@chef-builder:~#
```

Perhaps the previous list was obsoleted?

Reply:
Ah yes, that makes sense now, thanks